config/filter.d/sendmail-auth.conf

```diff
@@ -9,7 +9,8 @@ before = common.conf
 
 _daemon = (?:sendmail|sm-(?:mta|acceptingconnections))
 
-failregex = ^%(__prefix_line)s\w{14}: (\S+ )?\[(?:IPv6:<IP6>|<IP4>)\]( \(may be forged\))?: possible SMTP attack: command=AUTH, count=\d+$
+# "w{20}" will give support for IDs up to 20 characters long
+failregex = ^%(__prefix_line)s\w{20}: (\S+ )?\[(?:IPv6:<IP6>|<IP4>)\]( \(may be forged\))?: possible SMTP attack: command=AUTH, count=\d+$
```

Comment:
> up to 20 characters long...

Nope, it will match exactly 20 chars (so it will fail on all shorter IDs, and also in our test suite)... 
Something like `\w{14,20}` would be correct here.

Reply:
Also note that test cases are must, so please provide an example of log-line this should cover, or better extend this PR with a new test in [fail2ban/tests/files/logs/sendmail-auth](https://github.com/hvanmegen/fail2ban/edit/patch-1/fail2ban/tests/files/logs/sendmail-auth?pr=%2Ffail2ban%2Ffail2ban%2Fpull%2F2563) with JSON and line containing such ID.

Reply:
I've added a longline in the tests as well